examples/mqtt-service-basic-authentication/mqtt_service_basic_authentication.md

```diff
@@ -0,0 +1,18 @@
+# MQTT service - Basic authentication
+
+The `mqtt:Service` receives messages from the MQTT server using the `mqtt:Listener` via basic authentication. Basic authentication can be enabled by configuring the `username`, `password` in the `connectionConfig` of the `mqtt:ListenerConfiguration`. Use this to connect to a MQTT server secured with basic authentication.
+
+::: code mqtt_service_basic_authentication.bal :::
+
+## Prerequisites
+- Start a [MQTT broker](https://mqtt.org/software/) instance configured to use basic authentication.
+
+Run the program by executing the following command.
+
+::: out mqtt_service_basic_authentication.out :::
+
+>**Tip:** Run the MQTT client given in the [MQTT publisher - Basic authentication](/learn/by-example/mqtt-client-basic-authentication) example to produce some messages to the topic.
+
+## Related links
+- [`mqtt:ListenerConfiguration` record - API documentation](https://lib.ballerina.io/ballerina/mqtt/latest#ListenerConfiguration)
+- [MQTT service basic authentication - Specification](https://github.com/ballerina-platform/module-ballerina-mqtt/blob/main/docs/spec/spec.md#422-secure-listener)
```

Comment:
Should't we point to the ballerina spec page instead of github page ?

Reply:
check other places

Reply:
Yeah. Was following kafka and it is a ballerinax module. Will update

Reply:
Updated